ksqldb-cli/src/test/java/io/confluent/ksql/cli/console/ConsoleTest.testPrintSourceDescriptionWithExtractedHeader.approved.tabular

```diff
@@ -0,0 +1,43 @@
+
+Name                 : TestSource
+Type                 : TABLE
+Timestamp field      : 2000-01-01
+Key format           : kafka
+Value format         : avro
+Kafka topic          : kafka-topic (partitions: 1, replication: 1)
+Statement            : sql statement
+
+ Field  | Type                             
+-------------------------------------------
+ ROWKEY | VARCHAR(STRING)  (primary key)   
+ f_0    | BOOLEAN                          
+ f_1    | INTEGER                          
+ f_2    | BIGINT                           
+ f_3    | DOUBLE                           
+ f_4    | VARCHAR(STRING)                  
+ f_5    | ARRAY<VARCHAR(STRING)>           
+ f_6    | MAP<STRING, BIGINT>              
+ f_7    | STRUCT<a DOUBLE>                 
+ HEAD   | BYTES            (header('abc')) 
+-------------------------------------------
+
+Queries that read from this TABLE
+-----------------------------------
+readId (ERROR) : read query
```

Comment:
is this ERROR ok?

Reply:
Yeah, all the source description tests have errors to test printing errors